cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -31,6 +31,9 @@ spec:
   containers:
   - name: kube-proxy
     image: {{pillar['kube_docker_registry']}}/kube-proxy:{{pillar['kube-proxy_docker_tag']}}
+    resources:
+      requests:
```

Comment:
I don't want to curtail kube-proxy, I want it to get what it needs (plus I'm not 100% sure how much it will need). 

But I guess it doesn't make sense giving unlimited cpu to the proxy if nothing can run on the node, and if it bumps into the limit it's because something else with higher priority came along, and it'll just be slow, unlike memory.